config/config.go

```diff
@@ -932,7 +932,7 @@ type ConsensusConfig struct {
 	// height (this gives us a chance to receive some more precommits, even
 	// though we already have +2/3).
 	// NOTE: when modifying, make sure to update time_iota_ms genesis parameter
-	TimeoutCommit time.Duration `mapstructure:"timeout_commit"`
+	TargetRoundDuration time.Duration `mapstructure:"target_round_duration"`
```

Comment:
It's not really per round but per height because we only commit once per height. What this means of course is that any multiround height will likely be above 15 seconds and instantly jump to the next height meaning only 2/3 voting power will be included in the commit (i.e. there will be no time for straggler voters). We should be wary of this if we are rewarding validators on a per-height basis for voting involvement. Alternatively we could reward validators for involvement across 100 heights (instead of just one). Or we could introduce another parameter which states a minimum "wait for straggler votes" period. 

Actually, It might be fine if there's also a non-negligible time taken in `FinalizeBlock` and `PrepareProposal`. If it takes 1-2 seconds, then that's another 1 to 2 seconds that more votes (above the 2/3 line) can be received in. 

We also might not care to have commits with all signatures.

Conclusion: I think we should rename this variable to `TargetHeightDuration` and we should make sure we are measuring the percentage of voting power in commits.

Reply:
> It's not really per round but per height because we only commit once per height.

ahh I see! good catch. I wrongly assumed that since the start time is in `RoundState` it would get updated each round. Renamed as well

> We should be wary of this if we are rewarding validators on a per-height basis for voting involvement.

I was worried about this as well (briefly mentioned in top comment) as iirc, there was somewhere in the sdk where we're rewarding validators per signature. I looked again though and couldn't find anything, so it might be obfuscated. Regardless, the number of signatures included should be something that we collect data on in testground, and might even want to force this to occur if it isn't happening naturally.